python/batch-predictor/docker/env310.yaml

```diff
@@ -0,0 +1,8 @@
+name: merlin-model
+dependencies:
+  - python=3.10
+  - pip=22.2.2
```

Comment:
Why do you pin the pip version to 22.2.2?

Reply:
Older pips are missing wheels information for packages used by 3.10 or m1, resulting them to try to build from source which often fails. Newer pip is able to fix that. Anyway this is only used in our base image building. I have defaulted to old version for users environment